test/utils/deepObject.test.js

```diff
@@ -51,39 +51,41 @@ describe('[utils] deepObject', () => {
 		});
 		// the implementation is slower, becouse it can resolve mutch more combinations
 		// and loadash is optimize for re-runs with momory cache
-		it('[profiling] pathsToArray - '
-			+ 'should not significantly slower for string notations like lodash.toPath.', () => {
-			const testPath = 'ele0.ele1.ele2.ele3.ele4.ele5.ele6.ele7.ele8.ele9.ele10';
-			const iterations = 100000;
-			const slowerFactor = 10;
-
-			const s3 = Date.now();
-			for (let i = 0; i <= iterations; i += 1) {
-				testPath.split('.');
+		it(
+			'[profiling] pathsToArray - ' + 'should not significantly slower for string notations like lodash.toPath.',
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)